### DISCUSSION TITLE
Please Help Me! to tell me your favorite Yuru-chara(mascot)'s name

### DISCUSSION THREAD

**Discussion by 9002taicher** [NAIST-SE/AttractingContributors#118] [General]
Hi. I'm Taichi Taniguchi.
Please tell me your favorite yuru-chara(or mascot)'s name !

I'm looking for collaborators.

this is my git [link](https://github.com/9002taicher/AttractingContributors)

**Comment by KDR121** [comment]
I add Minamo.
plz help me if you can [this](https://github.com/KDR121/AttractingContributors).
